README.md

```diff
@@ -1,83 +1,100 @@
 # covidflow
 
-Flow for covid-19 assessment, navigation and follow-ups.
+Rasa backend for the covid-19 assessment, navigation and follow-ups available at https://covid19.dialogue.co.
 
-## Development
-
-### Make
-
-Make is expected to be installed on the developer workstation.
+## Folder Structure
```

Comment:
I would put this section below Prerequisites and Setup, maybe even below Makefile tasks

Reply:
since the "Makefile tasks" section refenreces some of the folders described here, I will move this section just before that.